pkg/hubble/metrics/api/context.go

```diff
@@ -128,8 +132,17 @@ func (cs ContextIdentifierList) String() string {
 type ContextOptions struct {
 	// Destination is the destination context to include in metrics
 	Destination ContextIdentifierList
+	// Destination is the destination context to include in metrics for egress traffic
+	DestinationEgress ContextIdentifierList
+	// Destination is the destination context to include in metrics for ingress traffic
+	DestinationIngress ContextIdentifierList
+
 	// Source is the source context to include in metrics
 	Source ContextIdentifierList
+	// Source is the source context to include in metrics for egress traffic
+	SourceEgress ContextIdentifierList
+	// Source is the source context to include in metrics for ingress traffic
+	SourceIngress ContextIdentifierList
```

Comment:
I think it would be useful to add information here in the comments about how:

1. `Source` and `Destination` set both ingress and egress.
2. `(Source|Destination)(Ingress|Egress)` override `Source` and `Destination`.

It is safe to assume that the above is implied without having to have it explicit, but this change might be helpful in clarifying how this works for folks just in case it isn't clear to them. Would prevent them from having to dig through the code to understand the logic involved here.

Reply:
done